egs/europarl_st/SRT/lcma_srt/decode/stage2/decode_hent_srt_m2o.sh

```
@@ -0,0 +1,130 @@
+    --encoder-dim-asr ${ASR_ENC_DIM} \
+    --encoder-unmasked-dim-asr ${ASR_UNMASK_DIM} \
+    --num-encoder-layers-st ${ST_NUM_LAYERS} \
+    --feedforward-dim-st ${ST_FF_DIM} \
+    --encoder-dim-st ${ST_ENC_DIM} \
+    --encoder-unmasked-dim-st ${ST_UNMASK_DIM} \
+    --downsampling-factor-st ${downsampling_factor_st} \
+    --cnn-module-kernel-st ${cnn_module_kernel_st} \
+    --num-heads-st ${num_heads_st} \
+    --downsampling-factor-asr ${downsampling_factor_asr} \
+    --chunk-size -1 \
+    --test-name $TEST_CUTS_PATH \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)